### ISSUE TITLE
How to improve the displayed precision for loss in the progress bar.

### ISSUE THREAD

**Issue by akum04** [PyTorchLightning/pytorch-lightning#5588]
During the training, I'm using the custom loss function to train my model. However my losses are displayed as 0.000, but when I display the same value to display as different variable it gives 4.73e-5 (some value with much better precision).
for example 
Epoch 80: 10%|██▌ | 100/1013 [01:33<14:11, 1.07it/s, loss=0.000, v_num=None, train_loss=4.73e-5]

both loss and train_loss display the same value. why one displays in exponential format and other doesn't.
Is it possible to improve the displayed precision for the loss in the progress bar.

**Comment by rohitgr7** [comment]
you can override `get_progress_bar_dict`
https://pytorch-lightning.readthedocs.io/en/latest/lightning_module.html#get-progress-bar-dict
https://github.com/PyTorchLightning/pytorch-lightning/blob/7abd8224d62c8bd3a0bb5b58c7d95488303daa98/pytorch_lightning/core/lightning.py#L1385

**Comment by akum04** [comment]
Thanks .... did override the function...It is working..
